config/initializers/valkyrie_characterization_service_override.rb

```diff
@@ -99,5 +103,16 @@ def digest_sha1
       end
       sha.hexdigest.prepend("urn:sha1:")
     end
+
+    def process_digest_preservation_event(file_set_id, user, event_start, value)
+      file_set = Hyrax.query_service.find_by(id: file_set_id)
+      # create event for digest calculation/failure
+      event = { 'type' => 'Message Digest Calculation', 'start' => event_start, 'details' => value,
+                'software_version' => 'FITS Servlet v1.6.0, Fedora v6.5, Ruby Digest library',
```

Comment:
For the software version, since it will change whenever we need update FITS/Fedora, could we set it somewhere (config, env) and pull the value from there, instead of setting a static value?

Reply:
@abelemlih Comments addressed with https://github.com/emory-libraries/dlp-selfdeposit/pull/232/commits/a63b189bbc3a6bffba5da0baf3bd8e45aa18a463